SmartDeviceLink/private/SDLPresentAlertOperation.m

```diff
@@ -193,8 +195,14 @@ - (void)sdl_uploadAudioFilesWithCompletionHandler:(void (^)(void))handler {
 /// @param handler Called when all images have been uploaded.
 - (void)sdl_uploadImagesWithCompletionHandler:(void (^)(void))handler {
     NSMutableArray<SDLArtwork *> *artworksToBeUploaded = [NSMutableArray array];
-    if ([self sdl_supportsAlertIcon] && [self.fileManager fileNeedsUpload:self.alertView.icon]) {
-        [artworksToBeUploaded addObject:self.alertView.icon];
+    if ([self sdl_supportsAlertIcon] && (self.alertView.icon != nil)) {
+        if ([self.fileManager fileNeedsUpload:self.alertView.icon]) {
+            // If the file is not uploaded, attempt to upload it
+            [artworksToBeUploaded addObject:self.alertView.icon];
+        } else if ([self.fileManager hasUploadedFile:self.alertView.icon] || self.alertView.icon.isStaticIcon) {
+            // If the file is already uploaded, add it to the uploaded set so we can show it
+            self.alertIconUploaded = YES;
+        }
```

Comment:
L204 is not covered in any of the unit tests. If possible should add or modify in unit tests to get full test coverage.